target-list-with-cloudflare/create-target-list.sh

```diff
@@ -7,17 +7,17 @@ FILE_PATH="<CF Real IP Conf Path>"  # /path/to/cf_real-ip.conf
 ip_addresses=$(grep -Eo '([0-9]{1,3}\.){3}[0-9]{1,3}(/[0-9]{1,2})?' $FILE_PATH | \
   grep -Ev '^172\.18\.')
 
-targets=$(echo -n "$ip_addresses"| jq -cRs 'split(" ")')
+targets=$(echo -n $ip_addresses | jq -cRs 'split(" ")')
 
 # Construct the JSON payload
 json_payload="{
-  \"name\": \"A Simple Target List with CloudFlare\",
+  \"name\": \"Target List with CloudFlare\",
   \"notes\":\"This is a Simple Target List with CloudFlare\",
   \"targets\": $targets
 }"
 
 # Make the API call
-http_code=$(curl -s -w %{http_code} --request POST \
+http_code=$(curl -s -w %{http_code} -o /dev/null --request POST \
```

Comment:
I assume the `<Firewalla Domain>` part will be replaced somewhere before this script can be executed, right ?